OpenUpSA/wazimap-ng-ui
# Applying number formatting to the choropleth info window

## Description
If choropleth is drawn for any sub-indicator the numbers and the percentage values on the popup are formatted according to the chart configuration of the same sub-indicator

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/143

## How to test it locally
* Set `hostname` parameter as `beta.youthexplorer.org.za` in index.js
* Select Youth poverty -> Multidimensional poverty -> Non poor on data mapper
* Hover over any geography
* Check if the numbers and the percentage values are formatted the same way as the chart of the same sub-indicator

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/101263823-6a4b4e00-3755-11eb-8dd6-b351cfc3c277.png)
![image](https://user-images.githubusercontent.com/53019884/101263926-746d4c80-3755-11eb-9c3e-7f4a37cf0a40.png)


## Changelog

### Added

### Updated
* `controller.js` to send the chart config through `subindicator` key of `state`. This key is used while creating the popups of the map
* `popup.js` to apply the formats to the numbers and the percentage values

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
